src/helpers/OTSubscriberHelper.js

```diff
@@ -89,16 +89,20 @@ const sanitizeFrameRate = (frameRate) => {
   }
 };
 
+const sanitizeScaleBehavior = (scaleBehavior = 'fill') => (scaleBehavior === 'fill' ? 'fill' : 'fit');
+
 const sanitizeProperties = (properties) => {
   if (typeof properties !== 'object') {
     return {
+      scaleBehavior: 'fill',
```

Comment:
I think having this as the default breaks it. I couldn't get it to work unless I changed this value here. I assume this is because the subscriber loads in a bit after the view is rendered?

Reply:
On which platform / or how does it break for you? 😯

Reply:
Both. The subscriber scaleBehavior is always fill

Reply:
![Screenshot 2021-05-27 at 11 54 07](https://user-images.githubusercontent.com/15106299/119824716-1ba19400-beee-11eb-8173-f37f23d9eed9.png)
![Screenshot 2021-05-27 at 11 56 58](https://user-images.githubusercontent.com/15106299/119824722-1d6b5780-beee-11eb-8b41-f6d10d7c97f5.png)